### PR TITLE
Removal of the WebSlice options

### LINKED ISSUE #3703: Removal of the WebSlice option
## Description of problem
Well, you know IE 8 technology and such, I think we could clean that up and remove it.

## Description of solution
Kill it :)

## Description of alternatives considered
Live in 1999

## Additional context
Since we no longer run tests for the browsers (well the one) that had support for this, I think we should remove it because we are not going to really support that anyway.

We must not forget to remove the localization keys relating to this feature too.

## Affected browser

* [ ] Chrome
* [ ] Firefox
* [ ] Safari
* [x] Internet Explorer
* [ ] Edge

### PR DESCRIPTION
**Pull request by joshgk00**
Fixes #3703

## Summary
The changes proposed here are to remove the WebSlice options from the Module Settings UI as well as the underlying tables, since it's no longer supported. 

## How to Test
- Do a clean install of DNN Platform and verify that the install completes without any errors.
- Upgrade an existing instance of DNN Platform and verify that the upgrade process completes without any errors
- Verify that the "Is a WebSlice" options have been removed from the Module Settings > Page Settings > Basic Settings section

Before this fix:
- Is a WebSlice options existed in Module Settings
image
- Views, Tables, Stored Procedures and Constraints referenced the IsWebSlice column in the TabModules table
   - TabModules Table
      image
   - TabModules Contraints
      image
   - TabModules Indexes
      image
   -  vw_TabModules
      image
   - SP: AddTabModule
     image
   - SP: UpdateTabModule
     image

After this fix:
- No options exist for the WebSlice Options in the Module Settings UI
image

- The Tables, Views, SPs, Indexes, and Constraints have been updated/removed

### CODE REVIEW

File: DNN Platform/Website/DotNetNuke.log4net.config [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (valadas) [inline DNN Platform/Website/DotNetNuke.log4net.config:1]: I think this file got committed by mistake right?

**Review by valadas — CHANGES_REQUESTED** [review]
Wow, awesome PRs, this shows why it's very easy to add things but hard to removed them :)

I think you really caught it all here, I just left 2 small comments. Great work!

File: DNN Platform/Library/UI/Modules/ModuleHost.cs [hunk 2]
```diff
@@ -250,14 +250,14 @@ private void InjectVersionToTheModuleIfSupported()
 
         private void InjectModuleContent(Control content)
         {
-            if (this._moduleConfiguration.IsWebSlice && !Globals.IsAdminControl())
+            if (!Globals.IsAdminControl())
```
[the remark this answers is not in the document]
Comment (joshgk00) [reply DNN Platform/Library/UI/Modules/ModuleHost.cs:253]: Yeah, I think you're right.

File: DNN Platform/Website/DotNetNuke.log4net.config (hunk 1 above)
Comment (joshgk00) [reply DNN Platform/Website/DotNetNuke.log4net.config:1]: Yes, this was not intentional.

**Review by valadas — APPROVED** [review]
Thanks for the quick turnaround on the requested changes. It all looks good to me now

**Comment by valadas** [comment]
Why do we have this one and #4710 @joshgk00 ? Should we close #4710 ?

**Comment by joshgk00** [comment]
I created a draft PR before I got the response that the changes should be pointed at the develop branch. #4710 can be closed out.

**Comment by valadas** [comment]
Awesome, I closed the other one.

**Comment by bdukes** [comment]
I've pinned this PR so it won't be marked as stale again

**Review by bdukes — APPROVED** [review]

**Pull request merged**